packages/@ember/-internals/glimmer/tests/integration/helpers/-parse-int-test.js

```diff
@@ -0,0 +1,14 @@
+import { RenderingTestCase, moduleFor } from 'internal-test-helpers';
+
+moduleFor(
+  'Helpers test: {{-parse-int}}',
+  class extends RenderingTestCase {
+    ['@test parseInt helper']() {
+      this.render(`{{-parse-int "123456789"}}`, {
+        someTruth: true,
+      });
+
+      this.assertTextNode(this.firstChild, '123456789');
+    }
+  }
+);
```

Comment:
Instead of having a unit test for the helper, I would prefer that we have an acceptance/integration test for being able to render/pass big numbers.

Reply:
https://github.com/emberjs/ember.js/blob/master/packages/@ember/-internals/glimmer/tests/integration/content-test.js would probably be a good place for it